### PR TITLE
Addition of computer systems researchers from University of Amsterdam (Fix for PR #5188)

### PR DESCRIPTION
**Pull request by adpimentel**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (up to three months, as processing has moved to a quarterly cadence).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR COMMIT MAY BE SUMMARILY REJECTED.

1. _Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv")._

1. _Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv`._

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

1. _Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files._ **Do not modify `csrankings.csv`, which is auto-generated.**

1. _Check to make sure that you have no spaces after commas, or any missing fields._

1. _Check to make sure the home page is correct._

1. _Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`)._

1. _Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org)._

1. _If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below)._

1. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [X] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

### CODE REVIEW

**Comment by adpimentel** [comment]
This is a fix for PR #5188.

File: csrankings-z.csv [hunk 1]
```diff
@@ -304,6 +305,8 @@ Zizhuo Wang 0001,CUHK (SZ),https://mypage.cuhk.edu.cn/academics/wangzizhuo/,bEJO
 Zohar Levi,Victoria University of Wellington,https://ecs.victoria.ac.nz/Main/ZoharLevi,kLGQWIsAAAAJ
 Zoltán Fülöp 0001,University of Szeged,http://www.inf.u-szeged.hu/~fulop,qnc6HewAAAAJ
 Zoltan Kato,University of Szeged,http://www.inf.u-szeged.hu/~kato,fgKiVvYAAAAJ
+Zoltan Mann,University of Amsterdam,https://cci-research.nl/author/zoltan-mann/,n73rycMAAAAJ
```
Comment (emeryberger) [inline csrankings-z.csv]: This alias does not appear in DBLP.

File: csrankings-f.csv [hunk 2]
```diff
@@ -421,6 +421,8 @@ Francesco Palmieri,University of Salerno,http://www.unisa.it/docenti/francescopa
 Francesco Palmieri 0001,University of Salerno,http://www.unisa.it/docenti/francescopalmieri/en/index,Ly2epXgAAAAJ
 Francesco Parisi-Presicce,Sapienza University of Rome,http://wwwusers.di.uniroma1.it/~parisi,BnSDUgwAAAAJ
 Francesco Ranzato,University of Padova,https://www.math.unipd.it/~ranzato,NvNaBzEAAAAJ
+Francesco Regazzoni,University of Amsterdam,https://cci-research.nl/author/francesco-regazzoni/,ERvZONQAAAAJ
```
Comment (emeryberger) [inline csrankings-f.csv]: This entry should not be here - it is not an alias.

**Review by emeryberger — CHANGES_REQUESTED** [review]
See comments.

**Comment by adpimentel** [comment]
Fixed!

Many thanks.

BR,
    Andy Pimentel

On 28 Nov 2022, at 00:17, Emery Berger ***@***.***> wrote:



@emeryberger requested changes on this pull request.

See comments.

________________________________

In csrankings-z.csv<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Femeryberger%2FCSrankings%2Fpull%2F5336%23discussion_r1033027653&data=05%7C01%7Ca.d.pimentel%40uva.nl%7C166285d2c687421afb1008dad0cd97fe%7Ca0f1cacd618c4403b94576fb3d6874e5%7C0%7C0%7C638051878685407626%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=uTNN%2FyXKbA%2F3Y%2Bfj4coLHgwW50xQkQgXeENPP69IUmk%3D&reserved=0>:

> @@ -304,6 +305,8 @@ Zizhuo Wang 0001,CUHK (SZ),https://mypage.cuhk.edu.cn/academics/wangzizhuo/,bEJO
 Zohar Levi,Victoria University of Wellington,https://ecs.victoria.ac.nz/Main/ZoharLevi,kLGQWIsAAAAJ
 Zoltán Fülöp 0001,University of Szeged,http://www.inf.u-szeged.hu/~fulop,qnc6HewAAAAJ
 Zoltan Kato,University of Szeged,http://www.inf.u-szeged.hu/~kato,fgKiVvYAAAAJ
+Zoltan Mann,University of Amsterdam,https://cci-research.nl/author/zoltan-mann/,n73rycMAAAAJ


This alias does not appear in DBLP.

________________________________

In csrankings-f.csv<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Femeryberger%2FCSrankings%2Fpull%2F5336%23discussion_r1033027783&data=05%7C01%7Ca.d.pimentel%40uva.nl%7C166285d2c687421afb1008dad0cd97fe%7Ca0f1cacd618c4403b94576fb3d6874e5%7C0%7C0%7C638051878685407626%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=85yArRKLueQAH3Bz4wS2kyrmAwwmryhk7nwdgJapuYM%3D&reserved=0>:

> @@ -421,6 +421,8 @@ Francesco Palmieri,University of Salerno,http://www.unisa.it/docenti/francescopa
 Francesco Palmieri 0001,University of Salerno,http://www.unisa.it/docenti/francescopalmieri/en/index,Ly2epXgAAAAJ
 Francesco Parisi-Presicce,Sapienza University of Rome,http://wwwusers.di.uniroma1.it/~parisi,BnSDUgwAAAAJ
 Francesco Ranzato,University of Padova,https://www.math.unipd.it/~ranzato,NvNaBzEAAAAJ
+Francesco Regazzoni,University of Amsterdam,https://cci-research.nl/author/francesco-regazzoni/,ERvZONQAAAAJ


This entry should not be here - it is not an alias.

—
Reply to this email directly, view it on GitHub<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Femeryberger%2FCSrankings%2Fpull%2F5336%23pullrequestreview-1195072028&data=05%7C01%7Ca.d.pimentel%40uva.nl%7C166285d2c687421afb1008dad0cd97fe%7Ca0f1cacd618c4403b94576fb3d6874e5%7C0%7C0%7C638051878685407626%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=xVhUdKHmFLmQ2H0IyP%2FjWAmhJD0Ic%2FW%2FVhyqvoRj9EQ%3D&reserved=0>, or unsubscribe<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAUSUKDACX5Q5YH555COVQDLWKPTZTANCNFSM6AAAAAASMX464Q&data=05%7C01%7Ca.d.pimentel%40uva.nl%7C166285d2c687421afb1008dad0cd97fe%7Ca0f1cacd618c4403b94576fb3d6874e5%7C0%7C0%7C638051878685563842%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Y9oDUcZEqEPZw%2BdtqafpM1PsJ661EXS1vwCJ6gKtYbs%3D&reserved=0>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

----
Andy D. Pimentel
Professor of Computer Science
Parallel Computing Systems group
University of Amsterdam
www.pcs-research.nl

**Pull request merged**